docs/src/pages.ts

```diff
@@ -14,6 +14,60 @@ const pages: readonly MuiPage[] = [
     },
   },
   { pathname: '/blog', title: 'Blog', icon: standardNavIcons.BookIcon },
+  {
+    pathname: '/x/react-data-grid-group',
+    title: 'Data Grid',
+    children: [
+      { pathname: '/x/react-data-grid', title: 'Overview' },
+      { pathname: '/x/react-data-grid/demo' },
+      {
+        pathname: '/x/react-data-grid/columns',
+        children: [
+          { pathname: '/x/react-data-grid/column-definition' },
+          { pathname: '/x/react-data-grid/column-dimensions' },
+          { pathname: '/x/react-data-grid/column-visibility' },
+        ],
+      },
+      {
+        pathname: '/x/react-data-grid/rows',
+        children: [
+          { pathname: '/x/react-data-grid/row-definition' },
+          { pathname: '/x/react-data-grid/row-updates' },
+          { pathname: '/x/react-data-grid/row-height' },
+        ],
+      },
+      { pathname: '/x/react-data-grid/editing' },
+      {
+        pathname: '/x/api/data-grid-group',
+        title: 'API Reference',
+        children: [
```

Comment:
It needs its own page, this breaks https://deploy-preview-38047--material-ui.netlify.app/versions/.

Reply:
I think this is there just to test the side nav ⎯ shouldn't really render a page?!

Reply:
To be clear, the navigation in the experiments folder is the same as in https://mui.com/versions/. We need to isolation them

Reply:
Uhm... I see! Yeah, isolating it would be good! Just pushed a commit that changes all the links just for the sake of testing the design/layout but I can roll it back before merging this PR so the docs-infra docs space doesn't suffer any harm. Is that good?